docker/base.Dockerfile

```diff
@@ -14,15 +14,25 @@ ARG ZENML_VERSION
 RUN pip install zenml${ZENML_VERSION:+==$ZENML_VERSION}
 
 FROM base AS server
+
 RUN pip install zenml${ZENML_VERSION:+==$ZENML_VERSION}[server]
 
 WORKDIR /zenml
 
-RUN mkdir -p .zenconfig/local_stores/default_zen_store
-
 ENV ZENML_CONFIG_PATH=/zenml/.zenconfig \
     ZENML_DEBUG=true \
     ZENML_ANALYTICS_OPT_IN=false
 
+ARG USERNAME=zenml
```

Comment:
These changes are intentionally added only for the server part of the image, correct?

Reply:
Correct ! Adding them to the base image breaks a lot of things, especially when running local orchestrators, because the localhost user and the docker user may have different UIDs/GIDs. Tackling that side of the story is not something I wanted to do in this PR.